File: app/src/main/java/com/fazziclay/opentoday/gui/interfaces/StorageEditsActions.java
package com.fazziclay.opentoday.gui.interfaces;

import com.fazziclay.opentoday.app.items.item.CycleListItem;
import com.fazziclay.opentoday.app.items.item.FilterGroupItem;
import com.fazziclay.opentoday.app.items.item.GroupItem;

public interface StorageEditsActions {
    void onGroupEdit(GroupItem groupItem);
    void onCycleListEdit(CycleListItem cycleListItem);
    void onFilterGroupEdit(FilterGroupItem filterGroupItem);
}

File: app/src/main/java/com/fazziclay/opentoday/gui/interfaces/StorageEditsActions.kt
package com.fazziclay.opentoday.gui.interfaces

import com.fazziclay.opentoday.app.items.item.CycleListItem
import com.fazziclay.opentoday.app.items.item.FilterGroupItem
import com.fazziclay.opentoday.app.items.item.GroupItem

interface StorageEditsActions {
    fun onGroupEdit(groupItem: GroupItem)
    fun onCycleListEdit(cycleListItem: CycleListItem)
    fun onFilterGroupEdit(filterGroupItem: FilterGroupItem)
}